tests/generation/test_utils.py

```diff
@@ -98,10 +98,21 @@ class GenerationTesterMixin:
 
     def _get_input_ids_and_config(self, batch_size=2):
         config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
-        input_ids = inputs_dict[self.input_name]
+        input_ids = inputs_dict.pop(self.input_name)[:batch_size, :]
+        _ = inputs_dict.pop("attention_mask", None)
```

Comment:
```suggestion
        inputs_dict.pop("attention_mask", None)
```
(nit: we don't need to store the output. the same applies to the lines below)